### ISSUE TITLE
Won't connect through company proxy

### ISSUE THREAD

**Issue by volsk** [stonesam92/ChitChat#113]
It seems ChitChat doesn't use my network proxy settings. As a result it won't connect when I am in the office, where a proxy is required to connect to the Internet.

**Comment by iDuuck** [comment]
Is it possible to browse to http://web.whatsapp.com/ ?

**Comment by volsk** [comment]
Yes. It is. Safari goes through directly. Chrome will ask for credentials
on the first outbound connection of the day.
